Prelude.playground/Sources/playgroundController.swift

```diff
@@ -34,10 +34,21 @@ public func playgroundControllers(device: Device = .phone4_7inch,
   -> (parent: UIViewController, child: UIViewController) {
 
     let parent = UIViewController()
-    parent.addChildViewController(child)
+
+    child.view.translatesAutoresizingMaskIntoConstraints = false
```

Comment:
Turns out Playgrounds don't like assigning `frame`s - therefore I've changed the logic to use Auto Layout